platform/mbed_assert.h

```diff
@@ -49,6 +49,65 @@ do {                                                     \
 } while (0)
 #endif
 
+
+#define _MBED_CONCAT_(a, b) a##b
+#define _MBED_CONCAT(a, b) _MBED_CONCAT_(a, b)
+#define _MBED_ASSERTION _MBED_CONCAT(_MBED_ASSERTION_AT_, __LINE__)
```

Comment:
`_MBED_CONCAT` and `_MBED_ASSERTION` are reserved identifiers according to the C standard: 

> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.